src/UseCases/GetInTouch/GetInTouchUseCase.php

```diff
@@ -24,17 +25,17 @@ public function __construct( GetInTouchValidator $validator, Messenger $messenge
 		$this->messenger = $messenger;
 	}
 
-	public function processContact( GetInTouchRequest $request ): string {
+	public function processContact( GetInTouchRequest $request ): ValidationResponse {
 		$this->request = $request;
 
 		if ( !$this->validator->validate( $request ) ) {
-			return 'validation failed';
+			return ValidationResponse::newFailureResponse( $this->validator->getConstraintViolations() );
 		}
 
 		try {
 			$this->forwardContactRequest();
 			$this->confirmToUser();
-			return 'request successful';
+			return ValidationResponse::newSuccessResponse();
 		} catch ( \RuntimeException $e ) {
 			return 'mail transmission failed';
```

Comment:
Sure, but to be very explicit, this would have to be a different response than `ValidationResponse` (because it's no validation error) and the route (or Presenter) would have to check for the different response types.